### DISCUSSION TITLE
Scripts to mount a borg backup and scan all the contends of /home/users for old email and import to mailpiler

### DISCUSSION THREAD

**Discussion by ediazrod** [jsuto/piler#175] [Show and tell]
On the backup server restore the copy what you want to use.

the uuid must be the same user than connect using sshfs

/programas/scripts/backup_borg/borg-linux64 mount -o uid=1000,gid=1000,allow_other /datos/backup_borg/backup::myservermailbackup-2020-12-31T02:00:03 /mnt/1

form you mailpiler install using root
sshfs uuiduser@servername.com:/mnt/1 /mnt/correo -o Ciphers=aes128-ctr -o Compression=no -o ServerAliveCountMax=2 -o ServerAliveInterval=15 

cd /var/tmp
find /mnt/correo/datos/ClonBorg/home/ -type f -name \*servername.com\* -exec sh -c 'gzip -dc "{}" | pilerimport -e -' \;

wait a few days and see on the logs how pilerimport injects the mails

After of that you can use another backup...

**Comment by jsuto** [comment]
Thanks for sharing!

**Reply by ediazrod** [reply]
Thanks to you to have this wonderfully software we need a donate button.. the gzip- -dc command is if you mails are compressed if is not you can delete gzip -dc "{}" |  and replace with a cat "{}"

**Reply by jsuto** [reply]
Good idea :-) I've just created a sponsorship button using [Buy Me Coffee](https://buymeacoffee.com/jsuto)
